pages/golfers/[id].js

```diff
@@ -0,0 +1,41 @@
+import Layout from '../../components/Layout'
+import ScorePostWidget from '../../components/ScorePostWidget'
+import ScoreCard from '../../components/ScoreCard'
+import useGolferScores from '../../lib/useGolferScores'
+import { useRouter } from 'next/router'
+
+const Golfer = () => {
+  const router = useRouter()
+  const { id } = router.query
+  const { scores, error } = useGolferScores(id)
+
+  if (!scores) {
+    return null
+  }
+
+  return (
+    <Layout>
+      <>
+        {error ? (
+          error
+        ) : (
+          <>
+            <ScorePostWidget />
```

Comment:
Let's show this widget only if we land on the page of the current user's scores.